SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -2122,6 +2116,13 @@ class CVariable {
    */
   inline su2double GetSolution_time_n(unsigned short val_var) {return Solution_time_n[val_var]; }
 
+  /*!
+   * \brief Get the solution at time n-1.
```

Comment:
If the use case for these new functions is that only the solver that instantiates the derived CVariable uses them, I suggest not declaring them in base CVariable all together and using the "special node" mechanism I am proposing in #753, it would keep the CVariable header a lot cleaner.